docker-compose.e2e-traversal.yml

```diff
@@ -360,14 +360,14 @@ services:
       - mongodb
     command: >
       -identityPassphrase ""
-      -myst 0x4D1d104AbD4F4351a0c51bE1e9CA0750BbCa1665
+      -Chain1Myst 0x4D1d104AbD4F4351a0c51bE1e9CA0750BbCa1665
```

Comment:
Updated commands but not bumped version here or am I just confused? 
```
  hermes2:
    image: mysteriumnetwork/hermes:0.7.0
```

Reply:
just opened a pr to run e2e to catch these sorts of issues, beats charging my laptop 🤷